README.md

```diff
@@ -1,183 +1,28 @@
-
-RINGMesh has previously been compiled with:
-
-* Visual Studio 10 2010 Win64
-* Visual Studio 11 2012 Win64
-* Visual Studio 12 2013 Win64
-* Visual Studio 14 2015 Win64
-* Visual Studio 15 2017 Win64
-
-Compiling RINGMesh
-------------------
-
-Make sure that you have installed C++ package for VisualStudio through the VisualStudio installer.
-You can either launch building in VisualStudio or calling cmake in command line
-in the build directory created at the configuration step:
-
-* cmake --build . --config Release
-* cmake --build . --config Debug
-* cmake --build . --config RelWithDebInfo
-
-The available compilation modes are:
-
-* Release
-* Debug
-* RelWithDebInfo (mandatory to debug a Gocad plugin in Debug mode with a Gocad
-  in Release, there are issues between libraries in Debug linked to a Gocad plugin)
-
-Compiling the documentation
----------------------------
-
-* Check the BUILD_DOCUMENTATION option when using cmake
-* Open the solution which is in build/RINGmesh.sln in VisualStudio
-* Build the doc-devkit or the doc-devkit-lite project
-
-See the documentation section for more details.
-
-Mac OS
-======
-
-Configuring RINGMesh
---------------------
-### Using clang (without Xcode)
-As in Linux.
-
-### Using Xcode IDE
-As in Windows but with the Xcode generator
-(use "-G Xcode" if you use cmake in command lines).
-
-Compiling RINGMesh
-------------------
-You need to install the Mac OS "Command Line Developer Tools".
-
-Note: you need gcc/g++ version higher or equal to 4.2 to compile RINGMesh.
-In Mac OS, clang is used.
-
-### Using clang (without Xcode)
-As in Linux except for the packages.
-
-### Using Xcode IDE
-You need to install Xcode IDE.
-Open the build/RINGMesh.xcodeproj with Xcode IDE,
-and then compile (as in Windows with VisualStudio).
-Or use these command lines:
-* cd build
-* xcodebuild -project RINGMesh.xcodeproj -alltargets -configuration Release
-
-To build in Debug, replace "Release" by "Debug" after "-configuration".
-
-Compiling the documentation
----------------------------
-
-See the documentation section for more details.
-
-### Using clang (without Xcode)
-As in Linux.
-
-### Using Xcode IDE
-As in Windows with VisualStudio but with Xcode.
-
-About documentation
-===================
+====================
 
-The documentation can be generated using [Doxygen](http://www.stack.nl/~dimitri/doxygen/).
-Two targets are available:
-* doc-devkit include full documentation of RINGMesh and Geogram
-* doc-devkit-lite include only the RINGMesh documentation
+RINGMesh is a cross platform project tested under Window (64bits), Linux (64bits) and Mac OS (64bits).  
+Please, follow the installation procedure described on the RINGMesh website: [ringmesh.org/try/installation/](http://ringmesh.org/try/installation/). 
 
-Then you can go in doc/devkit[-lite]/html and open the index.html with your web browser.
-A ringmesh.qch file is generated in doc/devkit[-lite]/html (to load in Qt Assistant).
+Develop with RINGMesh
```

Comment:
"Developing with RINGMesh" / "Using and developing RINGMesh" would be better I think.